misc/package/Data Files/MWSE/core/mwse/travelFinished/main.lua

```diff
@@ -0,0 +1,34 @@
+local timeBeforeTravel = 0
+local payload = {}
+
+---@param e cellChangedEventData
+local function travelEnded(e)
+	payload.cell = e.cell
+	payload.previousCell = e.previousCell
+	event.trigger(tes3.event.travelFinished, payload)
+	event.unregister(tes3.event.cellChanged, travelEnded)
+end
+
+---@param e calcTravelPriceEventData
+local function travelStart(e)
+	if (not tes3.mobilePlayer.traveling) then -- Get time before traveling
+		timeBeforeTravel = tes3.getSimulationTimestamp()
+	end
+
+	if tes3.mobilePlayer.traveling then	-- Travel finished, but calcTravelPrice triggered once again.
+		table.copy(e, payload)
+		payload.claim = nil
+		payload.hoursPassed = tes3.getSimulationTimestamp() - timeBeforeTravel
+		timeBeforeTravel = 0
+
+		-- calcTravelPrice event is triggered once for every possible destination on
+		-- first opening of the Travel Menu. Once again after a destination has been
+		-- selected, but before the player was teleported. This event is triggered
+		-- after the player is teleported to the destination.
+		event.register(tes3.event.cellChanged, travelEnded)
```

Comment:
This will not work if the travel destination is in the same cell, which could happen in a mod.
Perhaps the simulate event with {doOnce = true} can work there.
I would generally advise using doOnce if its going to only be executed once and then unregistered (unregister happens automatically in that case).

Reply:
Now I trigger the event a frame later<sup>tm</sup>, so it will be triggered even when the destination is in the same cell. Thanks for catching this! 

If anybody else wants to test this, go to balmora trademaster and there is a Balmora travel option with this [test plugin](https://github.com/MWSE/MWSE/files/9041986/Marker.test.zip).
 
.